RestaurantReservation/RestaurantReservation.Db/Models/Entities/Customer.cs

```diff
@@ -0,0 +1,11 @@
+namespace RestaurantReservation.Db.Models.Entities;
+
+public class Customer
+{
+    public int Id { get; set; }
+    public string FirstName { get; set; }
+    public string LastName { get; set; }
+    public string Email { get; set; }
+    public string PhoneNumber { get; set; }
+    public ICollection<Reservation>? Reservations { get; set; } = new List<Reservation>();
```

Comment:
* what about using **List<>**?
* try the new object initialization syntax 
```suggestion
    public ICollection<Reservation>? Reservations { get; set; } = new();
```

Reply:
Done